### PR TITLE
Adds some simple department signs

### PR DESCRIPTION
**Pull request by SodiumShine**
* Some departments didn't have any signs. So I added some.
* Also added a bathroom sign but the European kind because I don't like the other kind.
* Removed a Scavstation sign.

### CODE REVIEW

**Comment by SodiumShine** [comment]
why tho travis

File: code/game/objects/structures/signs.dm [hunk 1]
```diff
@@ -436,3 +431,68 @@
...
+	desc = "A sign labelled FORENSICS."
+	icon_state = "forensics"
+
+/obj/structure/sign/security
+	name = "\improper SECURITY"
+	desc = "A sign labelling the area as belonging to Security."
+	icon_state = "sec_scale"
+
+/obj/structure/sign/security/alt
+	name = "\improper SECURITY"
```
Comment (MistakeNot4892) [inline code/game/objects/structures/signs.dm]: These don't need to be set again as they are inherited from the previous definition.

Comment (SodiumShine) [reply code/game/objects/structures/signs.dm]: These? Do you just mean security/alt name or am I missing another?

Comment (MistakeNot4892) [reply code/game/objects/structures/signs.dm]: Name and desc on this one, I didn't see any others.

Comment (SodiumShine) [reply code/game/objects/structures/signs.dm]: Oh just realized mobile view is really bad and didn't make it clear what line you were pointing at.

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**